### ISSUE TITLE
Keyboard - Uncaught TypeError: Cannot call method 'getBoundingClientRect' of undefined

### ISSUE THREAD

**Issue by tobbbe** [driftyco/ionic#3586]
Sometimes when after creating an email with https://github.com/katzer/cordova-plugin-email-composer I get `Uncaught TypeError: Cannot call method 'getBoundingClientRect' of undefined`.

Here's more info: http://forum.ionicframework.com/t/cannot-read-property-getboundingclientrect-of-undefined-on-keyboard-show/21063

    function keyboardShow() {
        var elementBounds = keyboardActiveElement.getBoundingClientRect();

**Comment by Ionitron** [comment]
Greetings @tobbbe!

My sensors indicate that you need to update your issue through our [custom issue form](http://ionicframework.com/submit-issue/#/?iid=3586). We are now requiring all issues to be submitted this way, to ensure that we have all of the information necessary to fix them as quickly as possible.

**[Click Here To Update Your Issue](http://ionicframework.com/submit-issue/#/?iid=3586)**

I will have no choice but to close this issue if it is not resubmitted through the form. Please fill out the rest of the form, so that I may use my friendly robot powers to assist you.

Thank you!

<span ionitron-issue-resubmit></span>

**Comment by nillotrejos** [comment]
I'm getting the same error every time my keyboard opens. Not sure if its related, but it was working fine a couple days ago when I built the project before updating xcode from 6.3.0 to 6.3.1

**Comment by nillotrejos** [comment]
I just noticed something else. The error is gone when parent view is cached.
This is my scenario (it may help debugging):
I'm working on a messenger app, so every time I go into the messages view (child), send a new message and go back to to the conversation list view (parent), I need those conversations to be updated with a preview of the latest sent message. If the parent view is cached, that list is not updated when clicking back and cache-view="false" did the trick. Then I stumble upon a new bug, if the parent view is cached, trying to go back to the parent by swiping left gets my in a blank screen.
I removed the cache false in the parent view and the Uncaught TypeError was gone, so I'm stuck in point one again.
